src/guiapi/include/Rect16.h

```diff
@@ -18,11 +18,11 @@ enum class ShiftDir_t {
 
 ////////////////////////////////////////////////////////////////////////////////
 ///
-/// Class prepesents rectagle in graphics system where x rising to the right
+/// Class represents rectangle in graphics system where x rising to the right
```

Comment:
when improving EN wording, make sure you add a/the as well ;) .... (This) class represents A rectangle in A graphics system...